guides/common/modules/con_available-upgrade-paths.adoc

```diff
@@ -0,0 +1,39 @@
+[id="available-upgrade-paths"]
+= Available upgrade paths
```

Comment:
Are those really different paths?

To me it feels like it's always the same path:
1. ensure you run version X
2. upgrade server to X+1
3. upgrade the proxies to X+1

Sure, you can walk the last step at a different pace, but it's still the same path?

Compare to el8-to-el9 upgrade, where you can pick "in-place", "backup/restore", (and "rebuild") that use totally different technology, have different pro/contras yet ultimately get you to the same place ("system running on el9")

Having the overview here is good, the naming just somehow implies the user has a choice, while it's really limited.

Reply:
Renamed to "Upgrade path overview"